package.json

```diff
@@ -65,10 +65,10 @@
     "ipfs-api": "^3.0.1",
     "ipfs-block": "^0.3.0",
     "ipfs-block-service": "^0.3.0",
-    "ipfs-data-importing": "^0.3.3",
     "ipfs-merkle-dag": "^0.5.0",
     "ipfs-multipart": "^0.1.0",
     "ipfs-repo": "^0.7.1",
+    "ipfs-unixfs-engine": "^0.5.0",
```

Comment:
Better check the remaining deps as well. There are a bunch that are not at the latest